### PR TITLE
connect code of conduct to nav

### PR DESCRIPTION
**Pull request by JMStudiosJoe**
### :scroll: Description

- Describe the general shape of this PR (new feature? refactor? bug fix? one-line change?)
1 line change to include code of conduct navigation

- Describe what changes are being made
connect the page, not able to test locally Faraday default adapter issue

- Describe why these changes are being made
navigation chage to the _includes/navigation.html

- List the use cases and edge cases relevant to this PR
none?

- Describe how errors will be handled. How will we know if this code breaks in production
404 page not found if cant navigate

### CODE REVIEW

File: _includes/navigation.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <a href="./volunteer.html">Volunteer</a>
 <a href="./about.html">About Us</a>
 <a href="./projects.html">Projects</a>
+<a href="./codeofconduct.html">Code Of Conduct</a>
```
Comment (1ec5) [inline _includes/navigation.html:4]: If I’m not mistaken, this will appear in both the header and the footer. The additional link will cause the row of links in the header to get pretty cramped:

Header with Code of Conduct after Projects, no spacing between any link.

Would it be OK to only link to the code of conduct in the footer? Add the link right below this include directive:

https://github.com/codeforsanjose/codeforsanjose.github.io/blob/f1f56f4c9eb059fd93b1fdad6c3915ace8a0f7e7/_includes/footer.html#L4

(While you’re at it, the CoC capitalizes it as “Code of Conduct”, so we should be consistent with that spelling.)

Comment (JMStudiosJoe) [reply _includes/navigation.html:4]: got it
